ledger/trie.go

```diff
@@ -289,7 +281,7 @@ func (p *TrieProof) String() string {
 	interimIndex := 0
 	for j := 0; j < int(p.Steps); j++ {
 		// if bit is set
-		if p.Flags[j/8]&(1<<int(7-j%8)) != 0 {
```

Comment:
@tarakby is this change okey?

Reply:
`j`, `%` has an integer codomain, and `j %8` <= 7 by [specification](https://golang.org/ref/spec#Arithmetic_operators)

Reply:
That's correct, but we can replace this by `Bit(p.Falgs, j)` using this [tool](https://github.com/onflow/flow-go/blob/master/ledger/common/bitutils/utils.go#L6) function.

Reply:
Thanks for flagging @ramtinms. You're right I made changes in the past to the `Bit` function.